arm/.global/global.module.tests.ps1

```diff
@@ -787,6 +787,36 @@ Describe 'Deployment template tests' -Tag Template {
             $incorrectParameters | Should -BeNullOrEmpty
         }
 
+        It "[<moduleFolderName>] Conditional parameters' description should contain 'Required if' followed by the condition making the parameter required." -TestCases $deploymentFolderTestCases {
+            param(
+                $moduleFolderName,
+                $templateContent
+            )
+
+            if (-not $templateContent.parameters) {
+                # Skip test
+                $true | Should -Be $true
+                return
+            }
+
+            $incorrectParameters = @()
+            $templateParameters = $templateContent.parameters.Keys
+            foreach ($parameter in $templateParameters) {
+                $data = ($templateContent.parameters.$parameter.metadata).description
+                switch -regex ($data)
+                {
+                    '^Conditional. .*'
+                    {
+                        if ($data -notmatch '.*\. Required if .*') {
+                            $incorrectParameters += $parameter
+                        }
+                    }
+                    # Possible additional cases can be added here
```

Comment:
What other cases do you have in mind?

Reply:
Not yet, but in case something comes up we may use it there. Or not. What do you suggest?

Reply:
I assume you're hinting at (and I know this example doesn't make sense) things like "If a parameter is required, the description must contain xzy" etc.
If so, I'd suggest to either
- Have a new test case for each future case we may want to handle or
- Rename the current test to not call out specifically for the conditional

Either are absolutely fine by me. And if you meant something completly different, too - just wanted to point it out and align :)

Reply:
Removed the comment if confusing. Let me know if there is any other specific change you're suggesting for this test.